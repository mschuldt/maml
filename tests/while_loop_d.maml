arduino = Arduino(desktop=True)

@block(once)
def test():
    x = 0
    while x < 10:
        x = x + 1
        print(x)

    n = 0
    while n < 30:
        x = 0
        while x < 3:
            print(n+x)
            x = x + 1
        n = n + 10
    die()
    
arduino.send(test)
