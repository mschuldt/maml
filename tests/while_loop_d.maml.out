Initializing avm...
Ready.

1
2
3
4
5
6
7
8
9
10
0
1
2
10
11
12
20
21
22
bye.
